README.md

```diff
@@ -346,6 +346,13 @@ Contributions and suggestions for new features are welcome, as are bug reports!
 Please create a new [issue](https://github.com/ewels/rich-click/issues)
 or better still, dive right in with a pull-request.
 
+### Local setup
```

Comment:
It would be great to have a bit of prose text about the different code style linters - at least listing what we're using and what they do.

Reply:
Good idea. Added something.